### ISSUE TITLE
Unidentified speaker(s)

### ISSUE THREAD

**Issue by gclux** [clarin-eric/ParlaMint#79]
In the French corpus, utterances that cannot be associated with an identified speaker are assigned to a specific id. It is in general short sentences (exclamations, questions...) originating from a specific political group (or several), which can be identified by the transcriber notes (possibly only two speakers, described in the note, but this is rare). 

Here is an example of the current French encoding...
```
               <seg>
                  <kinesic type="applause">
                     <desc xml:lang="fr">Applaudissements sur les bancs du groupe LR</desc>
                  </kinesic>
               </seg>
            </u>
            <u who="#PA0"
               xml:id="ParlaMint-FR_2017-07-25-E1020_u17"
               ana="#unknown">
               <seg>Très bien !</seg>
            </u>

```            
As a result the id "PA0" must be defined in the metadata (currently with forename/surname set to "unknown").            

I am curious to understand how these situations are handled in other countries...

**Comment by matyaskopp** [comment]
The situation of anonymous/uncertain speaker has been discussed here: #29 and #33 

In CZ case - we consider as utterance only speeches that have been said to a microphone. 

Other vocal phenomena are not utterances, so we place them into utterances only if they somehow interrupt/disturb speech (note: `VOCALINCIDENT` is just pseudo-element):

```
<u>some text <VOCALINCIDENT/> that continues</u>
```

or we place them outside utterance if they are at the end of utterance (typically applause)

This situation does not appear in our data: 
```
<u><VOCALINCIDENT/></u>
```

**Comment by TomazErjavec** [comment]
> This situation does not appear in our data: `<u><VOCALINCIDENT/></u>`

I agree (of course) with @matyaskopp, with `<seg><VOCALINCIDENT/></seg>` being discouraged as well, because we then have elements without any proper transcription inside them (so, without `<s>` in .ana).

**Comment by gclux** [comment]
In the FR corpus, we prefer to keep the interruptions (not said at the microphone) as utterances in order to be accurate wrt the transcription (where we also get a code indicating this is an interruption, even though we don't have an attribute in ParlaMint to keep it... may be for a future use). Obviously, we should get rid of the dummy id given in the source.

About having the incidents outside, I understand it is the recommended solution. However, here as well there is an issue of accuracy (loyalty?) with the transcribers notes... Also, sometimes this may entail splitting a speech in several segments. The Parla-CLARIN specifications say they should be "inside the utterance" in this case....
https://clarin-eric.github.io/parla-clarin/#sec-incidents

**Comment by matyaskopp** [comment]
If you want to keep interruptions as utterances `u` you should treat them in the same way - every utterance must contain text.

You can probably introduce a new speaker role to distinguish the situation: `<u ana="#intruder"...>` (or some other better word)

**Comment by TomazErjavec** [comment]
I think this is now fixed, at least as much as possible for 2.1.
